### ISSUE TITLE
Disable the 'Max spot size' option in the 'Spot finding' tab of the options window until xia2 supports it

### ISSUE THREAD

**Issue by benjaminhwilliams** [xia2/i19-gui#7]
`max_spot_size=` is a valid parameter to pass to `dials.find_spots` but isn't recognised by xia2, unlike `min_spot_size=`.  Until this is resolved (see xia2/xia2#543), disable the option.

**Comment by graeme-winter** [comment]
You can work around this by your gui making a `find_spots.phil` and then passing this on the command line - I think there is another method for passing a phil parameter which is a phil file

**Comment by benjaminhwilliams** [comment]
Yes indeed, that's another possibility (identified in xia2/xia2#543).
